test/e2e/velero_utils.go

```diff
@@ -391,3 +393,93 @@ func waitForVSphereUploadCompletion(ctx context.Context, timeout time.Duration,
 
 	return err
 }
+
+func getVeleroVersion(ctx context.Context, veleroCLI string) (string, error) {
+	checkCMD := exec.CommandContext(ctx, veleroCLI, "version")
+
+	stdoutPipe, err := checkCMD.StdoutPipe()
+	if err != nil {
+		return "", err
+	}
+
+	jsonBuf := make([]byte, 1*1024)
+
+	err = checkCMD.Start()
```

Comment:
Use `Run` or `RunCommand` provided by velero library https://github.com/vmware-tanzu/velero/blob/main/test/e2e/install.go#L187 instead
`Start` starts the specified command but does not wait for it to complete

Reply:
It's fixed, thanks a lot.